products/nvm.md

```diff
@@ -17,6 +17,13 @@ auto:
       regex: ^v(?P<version>\d+\.\d+\.\d+)$
       template: "{{version}}"
 
+identifiers:
+  - repology: nvm
+  - purl: pkg:github/nvm-sh/nvm
+  - purl: pkg:brew/nvm
+  - purl: pkg:arch/arch/nvm
```

Comment:
these are not purls for nvm; i don't maintain it, and they do not have the same support policy. Specifically, I refuse to provide support for brew-installed nvm and nobody should be using it.